features/onboarding/hooks/use-signup-with-passkey.ts

```
@@ -1,67 +1,67 @@
         inboxSigner: signer,
-      });
+      })
 
       return {
         inboxId: xmtpClient.inboxId,
         ethereumAddress: smartWalletclient.account.address,
-      };
+      }
     } catch (error) {
-      throw error;
+      throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)